HW1/P2/P2.txt

```diff
@@ -1 +1,3 @@
-# Your discussion here
+In the first part, the partitioning strategy was to have adjacent points be on the same cluster. This is inefficient because if many points, all of which require a large amount of computation time, are all computed in the same RDD, then this will be slow. There will be RDD’s that are just sitting there, doing nothing while another one is still working. 
```

Comment:
I don't think there will be "RDD's doing nothing". If a partition has less work and finished early, the worker node will take the next partition in the queue. Your scenario may happen if the waiting queue is empty